### PR TITLE
[#149468479] Ch set up heroku

### PR DESCRIPTION
**Pull request by EdwinKato**
## PR Status
**READY**

## Story ID
#149468479.

## Added Migrations
NO

## Description
Deploy project to heroku.

## Related PRs
None

### CODE REVIEW

File: wger/settings.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
```
Comment (amwaleh) [inline wger/settings.py]: thought this was supposed to be auto generated? Wasnt this added to gitIgnore?

**Comment by SerryJohns** [comment]
@EdwinKato This pull request needs a description. Can you please follow the PR template.

File: wger/settings.py (hunk 1 above)
Comment (EdwinKato) [inline wger/settings.py]: I have updated and rectified this

**Comment by EdwinKato** [comment]
@SerryJohns Description has been added as requeted

**Pull request merged**